toolkit/tools/imagecustomizerapi/output_image.go

```diff
@@ -4,7 +4,8 @@
 package imagecustomizerapi
 
 type OutputImage struct {
-	Path string `yaml:"path" json:"path,omitempty"`
+	Path   string `yaml:"path" json:"path,omitempty"`
+	Format string `yaml:"format" json:"format,omitempty"`
```

Comment:
You should verify that format contains a valid value.

See, `FileSystemType` for an example.